media/js/firefox/whatsnew/whatsnew-62.js

```diff
@@ -0,0 +1,154 @@
+    };
+
+    Mozilla.WNP62.showPocket = function() {
+        // Show the content
+        this.mainContent.classList.add('show-pocket');
+
+        // swap out Mozilla logo for Pocket logo
+        this.pageHead.classList.add('show-pocket');
+
+        // Set the title
+        document.title = this.$strings.data('pocket-title');
+    };
+
+    Mozilla.WNP62.showFxa = function() {
+        // Show the content
+        this.mainContent.classList.add('show-fxa');
+
+        // Set the title
+        document.title = this.$strings.data('fxaccount-title');
+
+        // initialize the FxA iframe
+        this.client.getFirefoxDetails(function(data) {
+            Mozilla.FxaIframe.init({
+                distribution: data.distribution,
+                gaEventName: 'whatsnew-fxa'
+            });
+        });
+    };
+
+    Mozilla.WNP62.showFirefoxMobile = function() {
+        // Show the content
+        this.mainContent.classList.add('show-fx-mobile');
+
+        // Set the title
+        document.title = this.$strings.data('fxmobile-title');
+
+        var form = new Mozilla.SendYourself('send-firefox');
+        form.init();
+    };
+
+    Mozilla.WNP62.geolocate = function(callback) {
+        $.get('/country-code.json')
+            .done(function(data) {
+                if (data && data.country_code) {
+                    callback(data.country_code.toLowerCase());
+                } else {
+                    callback();
+                }
+            })
+            .fail(function() {
+                callback();
+            });
+    };
+
+    Mozilla.WNP62.showFocusOrKlar = function(countryCode, product) {
+        // Show the content
+        this.mainContent.classList.add('show-' + product);
+
+        // Set the title
+        document.title = this.$strings.data('fxfocus-title');
+
+        var form = new Mozilla.SendYourself('send-' + product);
+        form.init(countryCode);
+    };
+
+    Mozilla.WNP62.chooseFocusOrKlar = function(countryCode) {
+        var product = 'focus';
+
+        if (countryCode && this.klarCountryCodes.includes(countryCode)) {
+            product = 'klar';
+        }
+
+        this.showFocusOrKlar(countryCode, product);
+    };
+
+    Mozilla.WNP62.checkUpToDate = function() {
+        // bug 1419573 - only show "Your Firefox is up to date" if it's the latest version.
+        if (this.client.isFirefoxDesktop) {
+            this.client.getFirefoxDetails(function(data) {
+                if (data.isUpToDate) {
+                    document.querySelector('.page-header').classList.add('show-up-to-date-message');
+                }
+            });
+        }
+    };
+
+    Mozilla.WNP62.init = function(fxaDetails) {
+        this.checkUpToDate();
+
+        // if user is not signed in to FxA...
+        if (!fxaDetails.setup) {
+            // Show Pocket for English locales
+            if (['en', 'en-US', 'en-CA', 'en-GB', 'en-ZA'].indexOf(this.locale) >= 0) {
```

Comment:
So it seems en-US is just `lang="en"` in the actual DOM ... so really it's the `en-US` string we probably don't need, but I added it to be safe. I checked and we have no other Englishes besides these four. en-AU, en-NZ etc just go to en-US by default.

Reply:
Sorry, I misread... so only matching 'en' doesn't work because it wants a complete string, though I could do a regex to match the first two characters explicitly. But then I have to figure out regex :(

Reply:
It's weird that it wants a full string. indexOf == 0 should match anything starting with `en`

Reply:
Oh I see... it's testing for this locale in the array, not something in the array corresponding to the locale.

Reply:
It's fine to leave it this way.

Reply:
I'm sure it could be done the other way, this is just the what I pasted from stackoverflow when I searched for "javascript check if string is in a list" and it worked 😁